docs/src/reference/ROCMUtilities.rst

```diff
@@ -1,5 +1,12 @@
+.. meta::
+  :description: ROCm CMake
+  :keywords: ROCm, Cmake, library, api, AMD
+
+.. _rocmutilities:
+
+****************************************************
 ROCMUtilities
-=============
+****************************************************
 
```

Comment:
This page could use an intro - what does Internal Utilities mean?
By the way, why are we publishing this if only internal?